src/main/java/com/hbm/tileentity/machine/TileEntityCustomMachine.java

```diff
@@ -223,6 +228,24 @@ public void updateEntity() {
 				this.progress = 0;
 			}
 			this.networkPackNT(50);
+		} else {
+			float volume = this.getVolume(1F);
+			if(this.progress > 0) {
```

Comment:
Also perhaps not the best way of doing this, every recipe cycle there is one tick where the machine is running, but the progress is 0, so this would momentarily stop the sound and reset it every time the machine finishes a recipe, even if it is still running.